eva-server/src/main/java/uk/ac/ebi/eva/server/Profiles.java

```diff
@@ -0,0 +1,9 @@
+package uk.ac.ebi.eva.server;
+
+/**
+ * Created by jorizci on 01/11/16.
+ */
+public class Profiles {
+    public static final String OAUTH_SECURITY = "oauth2-security";
+    public static final String NOT_OAUTHSECURITY = "!"+ OAUTH_SECURITY;
```

Comment:
Right now we only have OAuth and non-OAuth, but in the future there will be more authentication schemas. Could this be renamed to something more user-friendly like "no-security" or "open"?